### ISSUE TITLE
[💡 Feature]: disable colored output in  @wdio/spec-reporter

### ISSUE THREAD

**Issue by rchl** [webdriverio/webdriverio#12051]
### Is your feature request related to a problem?

`@wdio/spec-reporter` reporter always includes shell escape codes to create colored text in the output. When viewing logs in an editor or anywhere else that is not a terminal those escape codes are just noise that make the output hard to read.

### Describe the solution you'd like.

Add option for disabling colored text.
Potentially even try to detect terminal capabilities and automatically disable colored output.

A new option like `color` could default to `true` or `"auto"` if it would support auto-detecting terminal capabilities.

### Describe alternatives you've considered.

Just deal with escape codes being there and making the output harder to read.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by christian-bromann** [comment]
@rchl thanks for raising the issue. We are using `chalk` for coloring the terminal so you can disable coloring via the `FORCE_COLOR` environment flag, e.g. `FORCE_COLOR=0 npx wdio run wdio.conf.js`. I pushed a commit to document this better.

**Comment by rchl** [comment]
It might not work for my use case. I have two `spec` reporters set up:

```
    reporters: [
        ['spec', { outputDir: './log/spec/', stdout: false, showPreface: false }],
        ['spec', { showPreface: false }],
    ],
```

One is supposed to print to the terminal and include colored output and the other one saves to files which are not supposed to have color.

**Comment by christian-bromann** [comment]
Oh, I see. Yes for this use case we need an option to the reporter. The way you've outlined it makes sense. Any contributions are appreciated!

**Comment by christian-bromann** [comment]
This has been resolved with the latest release [`v8.29.7`](https://github.com/webdriverio/webdriverio/commit/1c7195268663df56856494c69d7bd655c834a368). A `color` option is now available for the spec reporter.
